### PR TITLE
Migrate documentation to Read-The-Docs

### LINKED ISSUE #48: Move docs to readthedoc
Move the docs to read the docs like I did for Herbie.

### PR DESCRIPTION
**Pull request by blaylockbk**
Resolves #48

Readthedocs offers automatic building of the docs. This will help out a ton with maintaining the docs. (I've always manually built them).

### CODE REVIEW

**Comment by blaylockbk** [comment]
I'm currently getting this error
```
Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'error'
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [3 lines of output]
      setup.py:120: UserWarning: Unable to determine GEOS version. Ensure you have 3.7.2 or later installed, or installation may fail.
        '.'.join(str(v) for v in GEOS_MIN_VERSION), ))
      Proj 8.0.0 must be installed.
      [end of output]
```

Will need to follow up with this and how I did it for the Herbie docs; I'm missing something.

**Comment by blaylockbk** [comment]
Oh, nice. looks like the docs worked!

File: docs/conf.py [hunk 1]
```diff
@@ -98,7 +98,7 @@
             "name": "SynopticPy",
             "url": "https://blaylockbk.github.io/SynopticPy/_build/html/",
         },
-        {"name": "Herbie", "url": "https://blaylockbk.github.io/Herbie/_build/html/"},
+        {"name": "Herbie", "url": "https://herbie.readthedocs.io/"},
```
Comment (amotl) [inline docs/conf.py:101]: You would like to use a different project name here, right?

**Review by amotl — COMMENTED** [review]
I am happy to see that you are instantly starting to carry this over to your other excellent repositories. ❤️

File: docs/conf.py (hunk 1 above)
Comment (amotl) [reply docs/conf.py:101]: Oh nevermind. You are actually linking to Herbie here.

Comment (blaylockbk) [reply docs/conf.py:101]: Yep! It's super easy to apply this to my other repos now that I have a template to follow thanks to you. It'll take a few days to find all the links between my projects, but I'm working on it 😁

**Pull request merged**